### PR TITLE
Go to quick edit page from vsm pipeline edit if quick edit toggle is turned on

### PR DESCRIPTION
**Pull request by yankurk**
#3777

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/controllers/value_stream_map_controller.rb [hunk 1]
```diff
@@ -58,10 +58,17 @@ def render_vsm_json(vsm, result)
     stage_detail_path_partial = proc do |pipeline_name, pipeline_counter, stage_name, stage_counter|
        stage_detail_tab_path(:pipeline_name => pipeline_name, :pipeline_counter => pipeline_counter, :stage_name => stage_name, :stage_counter => stage_counter)
     end
-    pipeline_edit_path_proc = proc { |pipeline_name | pipeline_edit_path(:pipeline_name => pipeline_name, :current_tab => 'general') }
+    pipeline_edit_path_proc = proc { |pipeline_name | edit_path_for_vsm_pipeline(pipeline_name) }
     ValueStreamMapModel.new(vsm, result.message(localizer), localizer, vsm_path_partial, vsm_material_path_partial, stage_detail_path_partial, pipeline_edit_path_proc).to_json
   end
 
+  def edit_path_for_vsm_pipeline(pipeline_name)
+    if is_quick_edit_page_default?
```
Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/app/controllers/value_stream_map_controller.rb]: Test for this?

Comment (varshavaradarajan) [reply server/webapp/WEB-INF/rails.new/app/controllers/value_stream_map_controller.rb]: Make this a private method. Its only used to set the pipeline path in the ValueStreamMapModel.

**Comment by ketan** [comment]
@varshavaradarajan — there are log files merged as part of this PR. Can you be more careful next time?

**Comment by varshavaradarajan** [comment]
Oh, sorry.  I think there was a force push and I didn't check. Will do so next time.

**Pull request merged**